app/_src/gateway/migrate-to-new-dns-client.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Migrate to the new DNS client
+---
+This guide walks you through migrating from the old DNS client to the new one. The new DNS client introduces and new standardized way to configure a service, and helps improve performance.
+
+## Migrate the DNS client configuration
```

Comment:
I'm not super clear on the migration steps. Do I change the custom directives over and then enable the new DNS client?

Reply:
I guess you would probably do everything at the same time, I added this section first because you shouldn't enable the client before at least checking the rest of the config

Reply:
This makes sense to me then!